apps/mobile/src/state/storage-persistors.ts

```diff
@@ -11,27 +12,47 @@ export const persistConfig: PersistConfig<RootState> = {
   whitelist: ['wallets', 'accounts', 'keychains', 'settings'],
 };
 
+const secureStoreConfig: SecureStore.SecureStoreOptions = {
+  authenticationPrompt: "Allow Leather to access application's secure storage",
+  requireAuthentication: true,
+  keychainAccessible: SecureStore.WHEN_UNLOCKED_THIS_DEVICE_ONLY,
+};
+
+const mnemonicSchema = z.object({
+  mnemonic: z.string(),
+  passphrase: z.string().optional(),
+});
+export type Mnemonic = z.infer<typeof mnemonicSchema>;
+
+export async function deleteAllMnemonics(fingerprintArr: string[]) {
+  return await Promise.all(
+    fingerprintArr.map(async fingerprint => {
+      await mnemonicStore(fingerprint).deleteMnemonic();
+    })
+  );
+}
```

Comment:
```suggestion
export async function deleteAllMnemonics(fingerprints: string[]) {
  return await Promise.all(
    fingerprints.map(fingerprint => {
     mnemonicStore(fingerprint).deleteMnemonic();
    })
  );
}
```

In a Promise.all takes care of calling the promise, so the inner async/await isn't needed